tests/integration/test_end_to_end_dcp.py

```diff
@@ -137,16 +140,20 @@ class Test10xRun(TestEndToEndDCP):
 
     def test_10x_run(self):
         runner = self.ingest_store_and_analyze_dataset(dataset_fixture='10x')
-        try:
-            self.assertEqual(1, len(runner.primary_bundle_uuids))
-            self.assertEqual(1, len(runner.secondary_bundle_uuids))
-            expected_files = self.expected_results_bundle_files(runner.primary_bundle_uuids[0],
-                                                                self.CELLRANGER_10X_ANALYSIS_OUTPUT_FILES_REGEXES)
-            results_bundle_manifest = self.data_store.bundle_manifest(runner.secondary_bundle_uuids[0])
-
-            self.check_manifest_contains_exactly_these_files(results_bundle_manifest, expected_files)
-        finally:
-            runner.cleanup_primary_and_result_bundles()
+
+        with Timeout(2 * 3600) as to:
```

Comment:
Change to 1 hour and 50 mins here as well

Reply:
fixed